From 3297b8a18becf8780718a9830a857990848aa614 Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Tue, 31 May 2016 13:20:28 -0700
Subject: [PATCH 33/37] Fix documentation for DT_MAGIC types. (closes #3845)

With the rename of M_ to MUTT_, the documentation generator in
makedoc.c needed to be adjusted for the new prefix size.

Thanks to Damien Riegel for the suggested fix.
---
 doc/makedoc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/makedoc.c b/doc/makedoc.c
index 382b422..fb49af5 100644
--- a/doc/makedoc.c
+++ b/doc/makedoc.c
@@ -506,7 +506,7 @@ static void pretty_default (char *t, size_t l, const char *s, int type)
     case DT_MAGIC:
     {
       /* heuristic! */
-      strncpy (t, s + 2, l);
+      strncpy (t, s + 5, l);
       for (; *t; t++) *t = tolower ((unsigned char) *t);
       break;
     }
-- 
2.9.0

